images/ubuntu-slim/Dockerfile.build

```diff
@@ -21,23 +21,18 @@ RUN echo "Yes, do as I say!" | apt-get purge \
     libudev1 \
     tzdata
 
-RUN apt-get autoremove -y
-
+# replace shell bash with sh
 RUN sed -i 's/bash/sh/g' /etc/passwd
 
-WORKDIR /usr/share
-
-RUN tar -czf copyrights.tar.gz common-licenses doc/*/copyright
-
-RUN apt-get clean -y && \
-    gunzip copyrights.tar.gz && \
-    tar -rf copyrights.tar doc/*/copyright && \
-    gzip copyrights.tar && \
+# cleanup
+RUN apt-get autoremove -y && \
+    apt-get clean -y && \
+    tar -czf /usr/share/copyrights.tar.gz /usr/share/common-licenses /usr/share/doc/*/copyright && \
```

Comment:
This can be changed to `/usr/share/common-licenses*` so that `/usr/share/common-licenses` can be missing and `tar` doesn't complain about it. Makes this easier to fully reuse in other Dockerfiles